nuvla/job_engine/connector/kubernetes.py

```diff
@@ -132,6 +141,109 @@ def log(self, component: str, since: datetime, lines: int,
         return self.k8s.log(component, since, lines, namespace)
 
 
+class HelmAppMgmt(Connector, ABC):
+    """Class providing Helm application management functionalities using.
+    """
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.helm = Helm(_NUVLAEDGE_SHARED_PATH, **kwargs)
```

Comment:
Instead of `_NUVLAEDGE_SHARED_PATH`, `job.nuvlaedge_shared_path` should be used.

⚠️ Using this path will not work for helm deployment on a cloud k8s cluster (not a NE).
This class is used for both deployment on NE and on Cloud.

Reply:
Will be done.